flexmeasures/ui/utils/view_utils.py

```diff
@@ -30,7 +30,10 @@ def fall_back_to_flask_template(render_function):
     def wrapper(template_name, *args, **kwargs):
         try:
             return render_function(template_name, *args, **kwargs)
-        except Exception:  # noqa: B902
+        except Exception as e:
+            current_app.logger.error(
+                f"render_flexmeasures_template failed to load {template_name}, due to {e}."
```

Comment:
- Should this not be `render_function.__name__` instead of `render_flexmeasures_template`?
- Let's add "Rendering via Flask's render_template() ..." at the end of the

Reply:
New look:

```
[FLEXMEASURES][2025-01-03 10:46:21,503] WARNING: Rendering via Flask's render_template("error.html"). Failed to render via render_flexmeasures_template("error.html") due to (psycopg2.errors.InFailedSqlTransaction) current transaction is aborted, commands ignored until end of transaction block
```